src/shared/Layout/components/Layout.tsx

```diff
@@ -23,6 +23,8 @@ function Layout(props: LayoutProps): JSX.Element {
   const session = useAuth();
 
   useEffect(() => {
+    if (session === undefined) return;
```

Comment:
Unless I am mistaken 
```suggestion
    if (!session) return;
```

Reply:
ça va être la même chose ici